docs/concepts/storage-profiles-content.rst

```diff
@@ -0,0 +1,19 @@
+Flocker Storage Profiles are a way to help you choose different levels of service from your storage provider.
+For example, in a development environment you might only want a low cost storage option.
+However, if you need storage for a production environment, you can choose a high performance and more expensive option.
```

Comment:
[suggestion] `you can choose a high performance and more expensive option`. --> `you can choose a profile with high performance and high cost options.`  (making it clear that options exist within a profile)